bitnami/wavefront/README.md

```diff
@@ -0,0 +1,249 @@
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Parameters
+
+The following table lists the configurable parameters of the Wavefront chart and their default values.
+
+#### Global parameters
+
+| Parameter                 | Description                                                    | Default           |
+|---------------------------|----------------------------------------------------------------|-------------------|
+| `global.imageRegistry`    | Global Docker image registry                                   | `nil`             |
+| `global.imagePullSecrets` | Global Docker registry secret names as an array                | `[]` (does not add image pull secrets to deployed pods) |
+| `global.storageClass`     | Global storage class for dynamic provisioning                  | `nil`             |
+
+#### Common parameters
+
+| Parameter                    | Description                                                 | Default                                   |
+|------------------------------|-------------------------------------------------------------|-------------------------------------------|
+| `clusterName`                | Unique name for the Kubernetes cluster (required)           | `KUBERNETES_CLUSTER_NAME`                 |
+| `wavefront.url`              | Wavefront URL for your cluster (required)                   | `https://YOUR_CLUSTER.wavefront.com`      |
+| `wavefront.token`            | Wavefront API Token (required)                              | `YOUR_API_TOKEN`                          |
+| `commonLabels`               | Labels to add to all deployed objects                       | `{}`                                      |
+| `commonAnnotations`          | Annotations to add to all deployed objects                  | `{}`                                      |
+| `extraDeploy`                | Array of extra objects to deploy with the release           | `[]` (evaluated as a template)            |
+| `containerSecurityContext`   | Container security podSecurityContext                       | `{ runAsUser: 1001, runAsNonRoot: true }` |
+| `podSecurityContext`         | Pod security                                                | `{ fsGroup: 1001 }`                       |
+| `rbac.create`                | Create RBAC resources                                       | `true`                                    |
+| `serviceAccount.create`      | Create Wavefront service account                            | `true`                                    |
+| `serviceAccount.name`        | Name of Wavefront service account                           | `nil`                                     |
+| `kube-state-metrics.enabled` | Setup and enable Kube State Metrics for collection          | `false`                                   |
```

Comment:
Move this to a new section "Kube State Metrics parameters" so you can clearly see that it's a subchart